chirp/drivers/baofeng_uv17.py

```diff
@@ -149,7 +157,7 @@ class UV17(baofeng_uv17Pro.UV17Pro):
     upload_function = _upload
 
     MODES = ["FM", "NFM"]
-    BLOCK_ORDER = [16, 17, 18, 19,  24, 25, 26, 4, 6]
+    BLOCK_ORDER = [0x16, 0x17, 0x18, 0x19, 0x24, 0x25, 0x26, 0x4, 0x6]
```

Comment:
Erm, these are not equivalent. I'm guessing they should have been hex to begin with and your removal of L46 makes these proper? Even still L46 wasn't converting these from decimal to hex equivalents or anything... Just making sure you're intending the change here.

Reply:
It's cleaner this way. And makes the handling of corrupted memory blocks easier.

Reply:
I'm definitely not arguing that it's not cleaner, I just want to make sure you're accounting for it and I'm not sure I see where.

17 is not equal to 0x17, but it looks like you were multiplying this by 0x1000 above, but that hasn't changed to match this.